src/linear_feedback_controller_ros.cpp

```diff
@@ -268,6 +268,7 @@ return_type LinearFeedbackControllerRos::update_and_write_commands(
   if (!read_state_from_references()) {
     return return_type::ERROR;
   }
+  input_sensor_.joint_state.stamp = time;
```

Comment:
Why not simply?
```suggestion
  input_sensor_.header.stamp = time;
```

Reply:
`input_sensor` is of type `linear_feedback_controller_msgs::Eigen::Sensor` and it has no header.

Reply:
as `joint_state` you simply added it in this PR instead add it to sensor and control.
I would be much more consistent.

Reply:
More consistent with what?

Reply:
Also, mind that `control.header.stamp` would not have the same meaning as `joint_state.stamp`.
`control.header.stamp` should be the time when the control was produced, while `joint_state.stamp` is when the joint states were measured.